Extensions/xkit_patches.js

```diff
@@ -124,33 +124,74 @@ XKit.tools.parse_version = function(versionString) {
 	return version;
 };
 
+var blogs_get = document.createElement("script");
+blogs_get.textContent = "("+(function(){
+	var blogs = [];
+	var models = Tumblr.dashboardControls.allTumblelogs;
+
+	models.filter(function(model){
```

Comment:
Okay, I'm convinced that this is necessary. However, I believe this code should use `XKit.tools.add_function` and be inthe `XKit.extensions.xkit_patches.run` function. Ideally the postMessage listener would also be torn down in xkit_patches.destroy